src/components/Controls.vue

```diff
@@ -277,55 +281,58 @@ export default Vue.extend({
           />
 
           <v-row>
-            <v-col cols="5">
+            <v-col>
               <v-btn
-                class="px-2"
                 color="grey darken-3"
                 depressed
                 text
                 small
                 @click="releaseNodes"
               >
-                <v-icon small>
+                <v-icon
+                  left
+                  small
+                >
                   mdi-pin-off
                 </v-icon>
                 Release
               </v-btn>
             </v-col>
-
-            <v-col
-              cols="3"
-              class="px-0"
-            >
+            <v-spacer />
+            <v-col v-if="simulationRunning">
               <v-btn
-                class="ml-2 px-1"
                 color="primary"
                 depressed
                 small
-                @click="startSimulation"
+                @click="stopSimulation"
               >
-                <v-icon small>
-                  mdi-play
+                <v-icon
+                  left
+                  small
+                >
+                  mdi-stop
                 </v-icon>
-                Start
+                Stop
               </v-btn>
             </v-col>
 
             <v-col
-              cols="3"
-              class="px-0"
+              v-else
+              cols="6"
             >
               <v-btn
-                class="ml-4 px-1"
                 color="primary"
                 depressed
                 small
-                @click="stopSimulation"
+                @click="startSimulation"
               >
-                <v-icon small>
-                  mdi-stop
+                <v-icon
+                  left
+                  small
+                >
+                  mdi-play
                 </v-icon>
-                Stop
+                Start
               </v-btn>
```

Comment:
If the only difference between these buttons are the text, icon, and what function is called, then I think you would be better off unifying this into a single button that uses a computed prop to fill in the icon name and button text, and create a function `toggleSimulation()` that checks the value of `simulationRunning` and runs either `stopSimulation()` or `startSimulation()`. That will halve this code, and make it more readable.

I think this would really improve the code, but I'll leave it to your discretion.

Reply:
Fixed in 7f2055f